src/components/IndiaStateMap/IndiaMapStats.js

```diff
@@ -2,72 +2,68 @@ import React from 'react';
 import { useTranslation } from 'react-i18next';
 import { motion } from 'framer-motion';
 
-import * as population from '../../data/Population.json';
-
-function calculateCases(cases, mode, state) {
-  if (mode === 'normal') { return Number(cases); }
-  const casesPerMillion = Math.round((Number(cases) / population.default[state]) * 1000000);
-  return casesPerMillion.toLocaleString('en');
-}
-
 const IndiaMapStats = ({
   stats, handleAttributeClick, activeAttribute, mode, state,
 }) => {
   const { t } = useTranslation();
   const isNormal = mode === 'normal';
+  const confirmAttribute = isNormal ? 'confirmed' : 'confirmedPerMillion';
+  const activesAttribute = isNormal ? 'active' : 'activePerMillion';
+  const recoveredAttribute = isNormal ? 'recovered' : 'recoveredPerMillion';
+  const deathAttribute = isNormal ? 'deaths' : 'deathsPerMillion';
```

Comment:
Do we need to do this? We can keep the `activeAtrribute` separate and the `mode` separate. Because they can be changed exclusively. 

I mean when you change the `mode`, `activeAttribute` should remain the same and when you change `activeAttribute` the `mode` should not change.